src/ripple/ledger/PaymentSandbox.h

```diff
@@ -36,16 +36,26 @@ namespace detail {
 class DeferredCredits
 {
 public:
-    // Get the adjusted balance of main for the
-    // balance between main and other.
-    STAmount
-    adjustedBalance (AccountID const& main,
+    struct adjustment
+    {
+        adjustment (STAmount const& d, STAmount const& c, STAmount const& b)
+            : debits (d), credits (c), orgBalance (b) {}
+        STAmount debits;
```

Comment:
I don't know that there's a fix for this, but I find maintaining this kind of code (3 identically typed arguments in one constructor or function call) to be treacherous.  It's really easy to swap the order.  I wish I had an idea for a good fix.